RadarSDK/RadarAPIClient.m

```diff
@@ -1138,14 +1147,19 @@ - (void)geocodeAddress:(NSString *)query completionHandler:(RadarGeocodeAPICompl
                     }];
 }
 
-- (void)reverseGeocodeLocation:(CLLocation *)location completionHandler:(RadarGeocodeAPICompletionHandler)completionHandler {
+- (void)reverseGeocodeLocation:(CLLocation *)location 
+                        layers:(NSString *)layers
+             completionHandler:(RadarGeocodeAPICompletionHandler)completionHandler {
     NSString *publishableKey = [RadarSettings publishableKey];
     if (!publishableKey) {
         return completionHandler(RadarStatusErrorPublishableKey, nil, nil);
     }
 
     NSMutableString *queryString = [NSMutableString new];
     [queryString appendFormat:@"coordinates=%.06f,%.06f", location.coordinate.latitude, location.coordinate.longitude];
+    if (layers != NULL) {
+        [queryString appendFormat:@"layers=%@", layers];
+    }
```

Comment:
This will then be like:
```
    if (layers && layers.count > 0) {
        [queryString appendFormat:@"&layers=%@", [layers componentsJoinedByString:@","]];
    }
    ```